keystone-shell/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.6-alpine3.6
```

Comment:
Can't we just use the monasca/python.
It should have all musl and stack-fix included already AFAIR.